docs/source/dev/sphinx.rst

```diff
@@ -39,6 +39,18 @@ The following requirements need to be installed (once) to build our documentatio
     # python tools & style theme
     pip install -r requirements.txt # --user
 
+However, in order to not break any of your existing Python configurations you can also create a new environment that you only use for building the documentation.
```

Comment:
Required: there needs to be a comma in `... configurations, you can ...`
Optional: remove `However,`